daemon/cmd/daemon_main.go

```diff
@@ -1004,6 +1004,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Bool(option.EnableBGPControlPlaneStatusReport, true, "Enable the BGP control plane status reporting")
 	option.BindEnv(vp, option.EnableBGPControlPlaneStatusReport)
 
+	flags.String(option.BGPRouterIDAllocationMode, defaults.BGPRouterIDAllocationMode, "BGP router-id allocation mode.")
```

Comment:
It looks like the only 2 valid options here are "" or "default". Might be good to let users know what's expected here since anything else will cause an error.

Reply:
In my current implementation, even if user don't designate `BGPRouterIDAllocationMode`, `"default"` is set.
(If user explicitly designate `""`, router-id is not allocated and error occur.)
Is it undesirable? Should we accept `""` ?

Reply:
I wrote about restriction.